cmd/chartsvc/main.go

```diff
@@ -54,7 +54,7 @@ func setupRoutes() http.Handler {
 	apiv1.Methods("GET").Path("/charts/{repo}/{chartName}").Handler(WithParams(getChart))
 	apiv1.Methods("GET").Path("/charts/{repo}/{chartName}/versions").Handler(WithParams(listChartVersions))
 	apiv1.Methods("GET").Path("/charts/{repo}/{chartName}/versions/{version}").Handler(WithParams(getChartVersion))
-	apiv1.Methods("GET").Path("/assets/{repo}/{chartName}/logo-160x160-fit.png").Handler(WithParams(getChartIcon))
```

Comment:
I was going to keep the previous path for backward compatibility but as far as I understand it's not needed because the web clients check the information of the chart and get the icon URL from it.

Reply:
Perhaps we should just keep it around anyway, there might be cases where a client is not forming the URL from the API?

Reply:
okay, added just in case